src/lib.ts

```diff
@@ -2,11 +2,16 @@
 // Note: some any types are hard to remove here, because of the recusive types of the introspection
 import * as graphql from "https://esm.sh/graphql@16.5.0";
 
+import Ask from "https://deno.land/x/ask@1.0.6/mod.ts";
```

Comment:
Also I would like to avoid adding dependencies for such small purposes, if it can be avoided, will be easier to maintain.